### PR TITLE
Explain how to fix FOIT with Google fonts

### LINKED ISSUE #1901: Explain how to fix FOIT with Google fonts
A lot of developers use google fonts, so this may help them.

### PR DESCRIPTION
**Pull request by mauriciabad**
Fixes #1901.

Changes proposed in this pull request:

- Explain how to fix FOIT with Google fonts

### CODE REVIEW

**Comment by mauriciabad** [comment]
@googlebot I signed it!

File: src/site/content/en/lighthouse-performance/font-display/index.md [hunk 1]
```diff
@@ -49,6 +49,14 @@ Once the custom font is ready, it replaces the system font.
 (See the [Avoid invisible text during loading](/avoid-invisible-text) post
 for more information.)
 
+### Google fonts
```
Comment (kaycebasques) [inline src/site/content/en/lighthouse-performance/font-display/index.md]:
```suggestion
### Google Fonts
```

File: src/site/content/en/lighthouse-performance/font-display/index.md [hunk 2]
```diff
@@ -49,6 +49,14 @@ Once the custom font is ready, it replaces the system font.
...
+
+If you use Google fonts you can fix this by setting the attribute display to swap in your Google Fonts URL, like this:
```
Comment (kaycebasques) [inline src/site/content/en/lighthouse-performance/font-display/index.md]:
```suggestion
Add `&display=swap` to the end of your Google Fonts URL:
```

File: src/site/content/en/lighthouse-performance/font-display/index.md [hunk 3]
```diff
@@ -49,6 +49,14 @@ Once the custom font is ready, it replaces the system font.
...
+```html
+<link href="https://fonts.googleapis.com/css?family=Roboto:400,700&display=swap" rel="stylesheet">
+```
+Notice the **`&display=swap`** at the end of the url.
```
Comment (kaycebasques) [inline src/site/content/en/lighthouse-performance/font-display/index.md]:
```suggestion
```

**Comment by kaycebasques** [comment]
@mauriciabad thanks for the contribution, I have a couple of small requested changes

**Comment by kaycebasques** [comment]
@mauriciabad you need to click the **Commit Suggestion** button for each change I've requested 

image

**Comment by mauriciabad** [comment]
@googlebot I fixed it.

**Comment by mauriciabad** [comment]
@kaycebasques Done, is it ok now?

**Comment by kaycebasques** [comment]
Thank you, @mauriciabad!

**Pull request merged**